stack.py

```diff
@@ -0,0 +1,21 @@
+from linkedlist import LinkedList
+
+
+class Stack:
+    def __init__(self, values=None):
+        self.storage = LinkedList(values)
```

Comment:
Can you make storage private?
```suggestion
        self._storage = LinkedList(values)
```